### PR TITLE
Feature/map 72/add endpoint to get location

### PR DESCRIPTION
**Pull request by Yojuks**
Add schema and locations controller

### CODE REVIEW

File: backend/controllers/LocationsController.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import Location from "../models/Locations";
+
+const LocationController = {
+  async getLocationId(req: Request, res: Response) {
+    try {
+      const { id } = req.body;
+
+      const locations = await Location.findById(id);
+
+      return res.json(locations);
```
Comment (steadyGuy) [inline backend/controllers/LocationsController.ts]: Maybe, we need to add a use-case when "id" from user will be incorrect ore not exist? And then throw 400 error.

**Review by steadyGuy — APPROVED** [review]
Looks nice

File: backend/controllers/LocationsController.ts (hunk 1 above)
Comment (Yojuks) [reply backend/controllers/LocationsController.ts]: have fixed it

File: backend/controllers/LocationsController.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import { Response, Request } from "express";
+import Location from "../models/Locations";
+
+const LocationController = {
+  async getLocationId(req: Request, res: Response) {
```
Comment (6mahdapihka9) [inline backend/controllers/LocationsController.ts]: rename to getLocationById
(we are not getting not an id, but the location by it)

File: backend/routes/index.ts [hunk 3]
```diff
@@ -1,10 +1,13 @@
 import express from "express";
 import FilesController from "../controllers/FilesController";
 import UserController from "../controllers/UserController";
+import LocationController from "../controllers/LocationsController";
 
 const router = express.Router();
 
 router.get("/users", UserController.getUsers);
+router.get("/locations", LocationController.getLocationId);
```
Comment (6mahdapihka9) [inline backend/routes/index.ts]: method get does not have body, therefore method getLocationId can not get id prop from it.
solvation: use params like -
"/locations/:id"
and req.params['id'] in getLocationId method [row #7]

**Review by 6mahdapihka9 — CHANGES_REQUESTED** [review]

**Review by 6mahdapihka9 — APPROVED** [review]
seems good to me :) 👍

**Pull request merged**